# Logfile created on 2017-09-28 15:06:19 -0500 by logger.rb/54362
I, [2017-09-28T15:06:19.927327 #14442]  INFO -- : *** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
E, [2017-09-28T15:06:19.927487 #14442] ERROR -- : undefined method `process_auto_unassign' for #<Class:0x007fa4d129be18>
Did you mean?  process_escalation (NoMethodError)
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:90:in `eval'
(eval):1:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:90:in `eval'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:90:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:112:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:112:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:112:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:112:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:112:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:112:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:73:in `block in start_job'
I, [2017-09-28T15:06:19.928617 #14442]  INFO -- : *** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
E, [2017-09-28T15:06:19.929096 #14442] ERROR -- : stream closed (IOError)

E, [2017-09-28T15:06:19.929307 #14442] ERROR -- : failed to allocate memory (NoMemoryError)

E, [2017-09-28T15:06:19.929431 #14442] ERROR -- : stack level too deep (SystemStackError)

E, [2017-09-28T15:06:19.929808 #14442] ERROR -- : exception reentered (fatal)

E, [2017-09-28T15:06:19.946354 #14442] ERROR -- : undefined method `process_auto_unassign' for #<Class:0x007fa4d129be18>
Did you mean?  process_escalation (NoMethodError)
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:90:in `eval'
(eval):1:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:90:in `eval'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:90:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:112:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:112:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:112:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:112:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:73:in `block in start_job'
E, [2017-09-28T15:06:19.947242 #14442] ERROR -- : No such file or directory @ unlink_internal - /Users/newpooter/GitHub/zammad-1.5.0/tmp/cache_file_store_production/Permission%3A%3Aticket20170928-14442-8z5oxo (Errno::ENOENT)
/Users/newpooter/.rbenv/versions/2.3.1/lib/ruby/2.3.0/tempfile.rb:257:in `unlink'
/Users/newpooter/.rbenv/versions/2.3.1/lib/ruby/2.3.0/tempfile.rb:257:in `call'
/Users/newpooter/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:341:in `block in exception_log'
/Users/newpooter/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:340:in `each_object'
/Users/newpooter/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:340:in `exception_log'
/Users/newpooter/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:238:in `block (2 levels) in start_proc'
E, [2017-09-28T15:06:19.947385 #14442] ERROR -- : No such file or directory @ unlink_internal - /Users/newpooter/GitHub/zammad-1.5.0/tmp/cache_file_store_production/Ticket%3A%3AArticle%3A%3ASender%3A%3AAgent20170928-14442-129xba7 (Errno::ENOENT)
/Users/newpooter/.rbenv/versions/2.3.1/lib/ruby/2.3.0/tempfile.rb:257:in `unlink'
/Users/newpooter/.rbenv/versions/2.3.1/lib/ruby/2.3.0/tempfile.rb:257:in `call'
/Users/newpooter/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:341:in `block in exception_log'
/Users/newpooter/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:340:in `each_object'
/Users/newpooter/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:340:in `exception_log'
/Users/newpooter/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:238:in `block (2 levels) in start_proc'
E, [2017-09-28T15:06:19.947473 #14442] ERROR -- : No such file or directory @ rb_sysopen - /Users/newpooter/GitHub/zammad-1.5.0/tmp/pids/scheduler.log (Errno::ENOENT)
/Users/newpooter/.rbenv/versions/2.3.1/lib/ruby/2.3.0/logger.rb:703:in `initialize'
/Users/newpooter/.rbenv/versions/2.3.1/lib/ruby/2.3.0/logger.rb:703:in `open'
/Users/newpooter/.rbenv/versions/2.3.1/lib/ruby/2.3.0/logger.rb:703:in `open_logfile'
/Users/newpooter/.rbenv/versions/2.3.1/lib/ruby/2.3.0/logger.rb:695:in `set_dev'
/Users/newpooter/.rbenv/versions/2.3.1/lib/ruby/2.3.0/logger.rb:635:in `initialize'
/Users/newpooter/.rbenv/versions/2.3.1/lib/ruby/2.3.0/logger.rb:353:in `new'
/Users/newpooter/.rbenv/versions/2.3.1/lib/ruby/2.3.0/logger.rb:353:in `initialize'
/Users/newpooter/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:335:in `new'
/Users/newpooter/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:335:in `exception_log'
/Users/newpooter/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/daemons-1.2.4/lib/daemons/application.rb:238:in `block (2 levels) in start_proc'
E, [2017-09-28T15:06:19.947564 #14442] ERROR -- : STOP thread for Ticket.process_auto_unassign after 10 tries (RuntimeError)
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:114:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:112:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:112:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:112:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:112:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:112:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:112:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:112:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:112:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:112:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:73:in `block in start_job'
E, [2017-09-28T15:06:19.961953 #14442] ERROR -- : undefined method `process_auto_unassign' for #<Class:0x007fa4d129be18>
Did you mean?  process_escalation (NoMethodError)
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:90:in `eval'
(eval):1:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:90:in `eval'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:90:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:112:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:73:in `block in start_job'
E, [2017-09-28T15:06:19.964233 #14442] ERROR -- : undefined method `process_auto_unassign' for #<Class:0x007fa4d129be18>
Did you mean?  process_escalation (NoMethodError)
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:90:in `eval'
(eval):1:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:90:in `eval'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:90:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:112:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:112:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:112:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:73:in `block in start_job'
E, [2017-09-28T15:06:19.964959 #14442] ERROR -- : undefined method `process_auto_unassign' for #<Class:0x007fa4d129be18>
Did you mean?  process_escalation (NoMethodError)
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:90:in `eval'
(eval):1:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:90:in `eval'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:90:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:112:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:112:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:112:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:112:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:112:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:112:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:112:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:112:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:73:in `block in start_job'
E, [2017-09-28T15:06:19.965996 #14442] ERROR -- : undefined method `process_auto_unassign' for #<Class:0x007fa4d129be18>
Did you mean?  process_escalation (NoMethodError)
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:90:in `eval'
(eval):1:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:90:in `eval'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:90:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:112:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:112:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:73:in `block in start_job'
E, [2017-09-28T15:06:19.966251 #14442] ERROR -- : undefined method `process_auto_unassign' for #<Class:0x007fa4d129be18>
Did you mean?  process_escalation (NoMethodError)
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:90:in `eval'
(eval):1:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:90:in `eval'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:90:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:112:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:112:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:112:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:112:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:112:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:112:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:112:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:73:in `block in start_job'
E, [2017-09-28T15:06:19.967664 #14442] ERROR -- : undefined method `process_auto_unassign' for #<Class:0x007fa4d129be18>
Did you mean?  process_escalation (NoMethodError)
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:90:in `eval'
(eval):1:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:90:in `eval'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:90:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:112:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:112:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:112:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:112:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:112:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:112:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:112:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:112:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:112:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:73:in `block in start_job'
E, [2017-09-28T15:06:19.970102 #14442] ERROR -- : undefined method `process_auto_unassign' for #<Class:0x007fa4d129be18>
Did you mean?  process_escalation (NoMethodError)
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:90:in `eval'
(eval):1:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:90:in `eval'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:90:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:73:in `block in start_job'
E, [2017-09-28T15:06:19.970806 #14442] ERROR -- : undefined method `process_auto_unassign' for #<Class:0x007fa4d129be18>
Did you mean?  process_escalation (NoMethodError)
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:90:in `eval'
(eval):1:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:90:in `eval'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:90:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:112:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:112:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:112:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:112:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:112:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:73:in `block in start_job'
E, [2017-09-28T15:06:19.971057 #14442] ERROR -- : undefined method `process_auto_unassign' for #<Class:0x007fa4d129be18>
Did you mean?  process_escalation (NoMethodError)
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:90:in `eval'
(eval):1:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:90:in `eval'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:90:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:112:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:112:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:112:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:112:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:112:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:112:in `rescue in _start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:86:in `_start_job'
/Users/newpooter/GitHub/zammad-1.5.0/app/models/scheduler.rb:73:in `block in start_job'
